contracts/BridgeLimiter.sol

```diff
@@ -75,24 +79,30 @@ contract BridgeLimiter is
 
     /* ========== EXTERNAL FUNCTIONS ========== */
 
+    // there are duplicated calls to `calculateAmountInUSD`, one in `willAmountExceedLimit`
+    // and one in in line 100. Can we modify `willAmountExceedLimit` to let it take the USD amount instead?
     function updateBridgeTransfers(uint8 tokenId, uint256 amount) external override onlyOwner {
         require(amount > 0, "BridgeLimiter: amount must be greater than 0");
         require(
             !willAmountExceedLimit(tokenId, amount),
             "BridgeLimiter: amount exceeds rolling window limit"
         );
 
+        // We have this in a few places as well
         uint32 currentHour = uint32(block.timestamp / 1 hours);
 
         // garbage collect most recently expired hour if window is moving
+        // why do we need `hourlyTransferAmount[currentHour] == 0`?
         if (hourlyTransferAmount[currentHour] == 0 && oldestHourTimestamp < currentHour - 24) {
+            // are the parameters correct?
```

Comment:
we want to garbage collect the most recent hour to expire (the 25th hour away from the current hour). The first parameter is the starting hour to try and garbagecollect, the second hour is the last hour to try and garbagecollect.